freezer/freezer_url_test.go

```diff
@@ -14,6 +14,10 @@ import (
 func TestFreezerSink(t *testing.T) {
 	assert := assert.New(t)
 
+	strawOpen = func(url string) (straw.StreamStore, error) {
```

Comment:
Is this good practice in go to use function variables to help with testing?

Reply:
I don't think it's especially good or bad.  I think it's reasonable and I've seen it done elsewhere a few times, so it's not uncommon.